assets/src/edit-story/index.js

```diff
@@ -41,7 +42,9 @@ const initialize = (id, config, flags) => {
   Modal.setAppElement(appElement);
   render(
     <FlagsProvider features={flags}>
-      <App config={config} />
+      <ErrorBoundary>
```

Comment:
Can you also add the error boundary to `.storybook/stories/playground/index.js`?

Reply:
Actually, probably unnecessary